### PR TITLE
[WIP] russian ship vault

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl: 
:add: we received radar signals of space russian ships in the proximity of the station, being  withing 20 meters from them will violate the NAP and they will return fire
/:cl:

travis broke dont listen to him

pic

### CODE REVIEW

**Comment by DaxDupont** [comment]
more loot pinatas

**Comment by plapatin** [comment]
add a boombox that plays bandit radio 24/7 when turned on

**Comment by necromanceranne** [comment]
That coat and hat are robust as shit. And don't have any slowdown despite being spaceworthy. And you can craft it with durathread and bear pelts?

Okay...

**Comment by duckay** [comment]
nerf the stats* of the russian clothing  because i wanna craft it in game
*because they're too god damn high

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -364,3 +364,14 @@
 		if(prob(3))
 			M += pick(" Honh honh honh!"," Honh!"," Zut Alors!")
 	return trim(M)
+
+/obj/item/clothing/head/rus_hat
+	name = "russian hat"
+	desc = "An russian hat made out of space bear pelts, it seems spaceowrthy."
```
Comment (kingoftheautism) [inline code/modules/clothing/head/misc.dm]: spelling
```suggestion
	desc = "An russian hat made out of space bear pelts, it seems spaceworthy."
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -621,3 +621,14 @@
 	w_class = WEIGHT_CLASS_TINY
 	flags_inv = HIDEGLOVES|HIDEEARS|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
 	alternate_worn_layer = UNDER_HEAD_LAYER
+
+/obj/item/clothing/suit/rus_coat
+	name = "russian coat"
+	desc = "An russian coat made out of space bear pelts, it seems spaceowrthy."
```
Comment (kingoftheautism) [inline code/modules/clothing/suits/miscellaneous.dm]: spelling
```suggestion
	desc = "An russian coat made out of space bear pelts, it seems spaceworthy."
```

**Pull request closed without merging**